src/metrics/documentation_updated/index.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  SignedDataEvent,
+  MetricsSignature,
+  MetricData,
+} from "../../interfaces";
+import {
+  DocumentationUpdatedOutput,
+  DocumentationUpdatedPayload,
+} from "./interfaces";
+import octokit from "../../core/octokit";
+import { logger } from "../../core/logger";
+
+const MARKDOWN_FILE_EXTENSION = ".md";
+
+export const collectDocumentationUpdatedMetrics = async (
+  dataEvent: SignedDataEvent
+): Promise<MetricData> => {
+  const payload = dataEvent.payload as DocumentationUpdatedPayload;
+
+  const repo = payload.repository.name;
+  const owner = payload.repository.owner.login;
+  const number = payload.number;
+
+  let mdFilesChanged = 0;
+
+  try {
+    const request = await octokit.request(
+      "GET /repos/{owner}/{repo}/pulls/{pull_number}/files",
+      {
+        owner: owner,
+        repo: repo,
+        pull_number: payload.pull_request.number,
+      }
+    );
```

Comment:
I'm not sure how to do this exactly with octokit (just changing the URL template?), but we probably want to set a query parameter `per_page=100`. The default is 30, max is 100 according to the docs.

We could also add a comment that this might miss some .md files in very large PRs (or follow the pagination, but ehh)

Reply:
Maybe we should rather add a metric to check if PRs with more than 100 files occur instead of following the pagination, and then check if this happens regularly.